src/services/connection.service.ts

```diff
@@ -0,0 +1,221 @@
+import { logger } from "@blockr/blockr-logger";
+
+import { MessageType } from "../enums/messageType.enum";
+import { UnknownDestinationError } from "../exceptions/unknownDestinationError";
+import { IMessageListener } from "../interfaces/messageListener";
+import { RECIEVE_HANDLER_TYPE, RESPONSE_TYPE } from "../interfaces/peer";
+import { Message } from "../models/message.model";
+import { RoutingTable } from "../models/routingTable.model";
+import { Receiver } from "../services/receiver.service";
+import { Sender } from "../services/sender.service";
+import { DateManipulator } from "../util/dateManipulator";
+import { Deferred } from "../util/deffered.util";
+
+
+const MESSAGE_EXPIRATION_TIMER: number = 1;
+const MESSAGE_HISTORY_CLEANUP_TIMER: number = 60000; // One minute
```

Comment:
Please change `60000` to `60_000`.

Please also change the naming of both the consts, to:

MESSAGE_EXPIRATION_TIMER_IN_MIN: number = 1;
const MESSAGE_HISTORY_CLEANUP_TIMER_IN_MS: number = 60_000;